api/logs/permissions.py

```diff
@@ -12,8 +12,14 @@ class ContributorOrPublicForLogs(permissions.BasePermission):
     def has_object_permission(self, request, view, obj):
         assert isinstance(obj, (NodeLog)), 'obj must be a NodeLog, got {}'.format(obj)
 
-        for node_id in obj._backrefs['logged']['node']['logs']:
-            node = get_object_or_error(Node, node_id, display_name='node')
-            if ContributorOrPublic().has_object_permission(request, view, node):
+        if obj._backrefs.get('logged'):
```

Comment:
Oh, it looks like it's from a commit that was merged into `caneruguz/dashboard-redesign-2` Can you please bring this up to date with that branch @pattisdr?